src/coreclr/vm/gcenv.ee.cpp

```diff
@@ -1656,6 +1674,9 @@ void GCToEEInterface::AnalyzeSurvivorsFinished(size_t gcIndex, int condemnedGene
 {
     LIMITED_METHOD_CONTRACT;
 
+    // This callback is called at least once per run of the GC, so we can reset the work stealing infra in the MoveableGCPointerTracker here
```

Comment:
David asked me for a place that guarantees different GCs each time this method is called so this is a good place to do that. the comment is actually incorrect - this callback is called exactly once in a blocking GC.

it's unlikely this will change - it hasn't changed in many years and I have no plans to change it. and there's no reason to call this more than once in a GC. this is already used for multiple things like generational aware analysis and sending notification to the debugger.

Reply:
Should we rename it to something more appropriate that better describes what it is used for?